driverapi/driverapi.go

```diff
@@ -72,6 +72,11 @@ type Driver interface {
 	// only invoked for the global scope driver.
 	EventNotify(event EventType, nid string, tableName string, key string, value []byte)
 
+	// DecodeTableEntry passes the driver a key, value pair from table it registered
+	// with libnetwork and expects the driver to return a {endpoint ID, info string}
```

Comment:
`the options can be ENDPOINT_OBJECT, NETWORK_OBJECT, OPAQUE.` - I think this will be quite useful. For ex: overlay driver can create a table for NETWORK_OBJECT, which it can use to store the network level info, for ex: vxlan id. So in the network inspect verbose we can present the vxlan id as well. https://github.com/docker/docker/issues/31559